doc/src/compute_chunk_atom.rst

```diff
@@ -622,14 +622,16 @@ cylinder, x for a y-axis cylinder, and x for a z-axis cylinder.
 
 **Output info:**
 
-This compute calculates a per-atom vector, which can be accessed by
-any command that uses per-atom values from a compute as input.  See
-the :doc:`Howto output <Howto_output>` doc page for an overview of
+This compute calculates a per-atom vector (the chunk ID), which can
+be accessed by any command that uses per-atom values from a compute
+as input.  It also calculates a global scalar (the number of chunks),
+which can be similarly accessed everywhere outside of a per-atom context.
+See the :doc:`Howto output <Howto_output>` doc page for an overview of
```

Comment:
Can we add a sentence or two (similar to what is in the PR description) about how to disambiguate the scalar from the vector by using an equal-style variable.  I think anyplace we have a compute (or fix) producing multiple flavors of output (global, per atom, local), we should do this.  At the moment I think it is rare?  Probably a line or two in the Section howto on output would also be appropriate.

Reply:
I would say this description already explains the disambiguation in a compact but sufficient form. More detail would just needlessly add redundant information. We have too much of that in the manual already. There is a link to the general information right in the same paragraph. If we need to copy everything everywhere instead of referring to pages explaining common properties, this will become even more of a nightmare (and a haven for inconsistencies) as it already is.

But I agree than an explanation about how to use the scalar value in a per-atom context should be added to the howto document, as this really is not a property of the compute itself, but a general question of how to handle such situations. I just added a paragraph for that.